bundles/aliyun/src/main/java/org/apache/gravitino/oss/credential/OSSTokenProvider.java

```diff
@@ -166,6 +167,8 @@ private String createPolicy(Set<String> readLocations, Set<String> writeLocation
                       Statement.builder()
                           .effect(Effect.ALLOW)
                           .addAction("oss:GetBucketLocation")
+                          // OSS Hadoop connector need to get bucket information
```

Comment:
I didn't mean this line. I meant the line 163.
Other developers (including me) would like to know that the `GetBucketLocation` action is not sufficient. In other words, this is two-actions in a row is by design, not a careless mistake.

Reply:
OH, got it, update the comment and variable name